pkg/envdir/envdir.go

```diff
@@ -28,10 +29,18 @@ func EnvDir(dir string) ([]string, error) {
 	if fi, err := os.Stat(dir); os.IsNotExist(err) || !fi.IsDir() {
 		return nil, nil
 	}
-	fis, err := ioutil.ReadDir(dir)
+	entries, err := os.ReadDir(dir)
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to read env from dir:"+dir)
 	}
+	fis := make([]fs.FileInfo, 0, len(entries))
+	for _, entry := range entries {
+		fi, err := entry.Info()
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to read info from entry:"+entry.Name())
+		}
+		fis = append(fis, fi)
+	}
```

Comment:
Can you explain why we're making this change? I don't quite understand.

Reply:
This deprecation note explains it - https://pkg.go.dev/io/ioutil#ReadDir

Reply:
However I blindly converted. After looking at the code more, I can probably use the response from ReadDir directly.